src/mmw/js/src/compare/controllers.js

```diff
@@ -16,28 +18,87 @@ var CompareController = {
             App.currProject
                 .fetch()
                 .done(function() {
+                    setupProjectCopy();
```

Comment:
> Consider asking the user to log in before going to compare view. I may have missed some discussion around this, but I don't see much value in not saving the project by this point.

Unfortunately, we're on the hook to allow a full worflow for unauthenticated users.